builtin/game/item.lua

```diff
@@ -530,6 +530,18 @@ function core.node_dig(pos, node, digger)
 	end
 end
 
+function core.itemstring_with_palette(item, palette_index)
+	local stack = ItemStack(item) -- convert to ItemStack
```

Comment:
This function will not be called often. It is made for craft recipe registration.